ts/screens/profile/ProfileMainScreen.tsx

```diff
@@ -299,12 +269,6 @@ class ProfileMainScreen extends React.PureComponent<Props> {
               sectionHeader={I18n.t("profile.main.developersSectionHeader")}
             />
 
-            {this.developerListItem(
-              I18n.t("profile.main.experimentalFeatures.confirmTitle"),
-              this.props.isExperimentalFeaturesEnabled,
-              this.onExperimentalFeaturesToggle
-            )}
```

Comment:
Since no experimental features are available at now, we could:
- just hide the switch from the UI instead of deleting it. As we maintain the `isExperimentalFeatureEnabled` into the store, it could be useful for testing/comparing changes applied in the future.
- remove any reference to experimental features by removing also the `isExperimentalFeatureEnabled` state and the related items still present into the code.

Reply:
First solution could be better